pkg/apis/application/v1alpha1/generated.proto

```diff
@@ -177,6 +181,20 @@ message HealthStatus {
   optional string statusDetails = 2;
 }
 
+// Operation contains requested operation parameters.
+message Operation {
+  optional SyncOperation sync = 2;
+}
+
+// OperationState contains information about state of currently performing operation on application.
+message OperationState {
+  optional string status = 1;
+
+  optional string errorDetails = 2;
+
+  optional SyncOperationResult sync = 3;
```

Comment:
I think we may need RollbackOperationResult as well, if we have separate operations.

Reply:
Done